### PR TITLE
Metrics descriotions

### PR DESCRIPTION
**Pull request by bdfinst**
I had several requests from the show and tell attendees to get this out, so here you go.

### CODE REVIEW

File: docs/playbooks/metrics/average-build-downtime.md [hunk 1]
```diff
@@ -10,14 +11,16 @@ The average length of time between when a build breaks and when it is fixed.
 
 ### What is the intended behavior?
 
-Decrease the average time to recover from a broken build.
+Keep the pipelines always deployable by fixing broken builds as rapidly as possible. Proken builds are the highest priority since
```
Comment (jgetzie) [inline docs/playbooks/metrics/average-build-downtime.md]: Fix spelling "Proken builds" to "Broken builds"

File: docs/playbooks/metrics/change-fail-rate.md [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
 # {{ page.title }}
 
-The percentage of changes that result in negative customer impact, system degradation, or rollback.
+The percentage of changes that result in negative customer impact, or rollback.
 
-`changeFailRate = impactingChangeCount / changeCount`
+`changeFailRate = failedChangeCount / changeCount`
 
-"Software delivery performance is a combination of three metrics: [lead time](./development-cycle-time.html), [release frequency](./deploy-frequency.html), and [MTTR](./mean-time-to-repair.html). Change fail rate is not included, though it is highly correlated."
+"Software delivery performance is a combination of three metrics: [lead time](./development-cycle-time.html), [release frequency](./release-frequency.html), and [MTTR](./mean-time-to-repair.html). Change fail rate is not included, though it is highly correlated."
```
Comment (jgetzie) [inline docs/playbooks/metrics/change-fail-rate.md:14]: This statement is referenced in multiple locations, we should consider an include for Object Orientation.

Comment (bdfinst) [reply docs/playbooks/metrics/change-fail-rate.md:14]: Was debating that. Didn't want to deal with changing paths before coffee. I'm debating having there at all. It's there for a reason internally but may not matter here. Let's demo and get feedback.

**Review by jgetzie — APPROVED** [review]
Looks good to me!

**Pull request merged**